src/main/java/InsertionSort/InsertionSort.java

```diff
@@ -0,0 +1,19 @@
+package InsertionSort;
+
+public class InsertionSort {
+
+//got inspiration for this code from (https://www.geeksforgeeks.org/insertion-sort/)
+    public static int[] InsertionSort(int[] list) {
+        for(int i = 0; i < list.length - 1; i++) {
+            int j = 0;

```

Comment:
Why are you defining j inside your loop? This makes your O(n) in space. See yesterday's code review in the first part of lecture.